sceptre/bridge/config/prod/bridge-ops.yaml

```diff
@@ -0,0 +1,10 @@
+template:
+  type: http
+  url: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/master/templates/EC2/basic-ec2.j2
+stack_name: bridge-ops
+dependencies:
+  - prod/bootstrap.yaml
+parameters:
+  InstanceType: t3a.micro
```

Comment:
`t3a.micro` probably won't work because that's for ARM.  The template is used is [targeting intel (x86) AMIs](https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/EC2/basic-ec2.j2#L26).  You'll probably want to change to `t3.micro`

Reply:
Fixed, thanks!